devices/wiser/fuga_socket_outlet.json

```diff
@@ -0,0 +1,281 @@
+{
+	"schema": "devcap1.schema.json",
+	"manufacturername": "Schneider Electric",
+	"modelid": "LK/OUTLET/1",
+	"product": "LK/OUTLET/1",
+	"sleeper": false,
+	"status": "Bronze",
+	"path": "/devices/wiser/fuga_socket_outlet.json",
+	"subdevices": [
+		{
+			"type": "$TYPE_ON_OFF_PLUGIN_UNIT",
+			"restapi": "/lights",
+			"uuid": [
+				"$address.ext",
+				"0x06"
+			],
+			"items": [
+				{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion"
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "state/alert",
+					"description": "The currently active alert effect.",
+					"default": "none"
+				},
+				{
+					"name": "state/on",
+					"refresh.interval": 5,
+					"read": {
+						"at": "0x0000",
+						"cl": "0x0006",
+						"ep": 6,
+						"fn": "zcl"
+					},
+					"parse": {
+						"at": "0x0000",
+						"cl": "0x0006",
+						"ep": 6,
+						"eval": "Item.val = Attr.val",
+						"fn": "zcl"
+					}
+				},
+				{
+					"name": "state/reachable"
+				}
+			]
+		},
+		{
+			"type": "$TYPE_POWER_SENSOR",
+			"restapi": "/sensors",
+			"uuid": [
+				"$address.ext",
+				"0x06",
+				"0x0b04"
+			],
+			"items": [
+				{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion"
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "config/on"
+				},
+				{
+					"name": "config/reachable"
+				},
```

Comment:
I'm in doubt here. @Smanar - do you know how sensors should be defined? Currently when the device is not reachable, it will be reported like this:

**/lights/3**

```
    "state": {
        "alert": "none",
        "on": false,
        "reachable": false
    },
```

**/sensors/3**

```
{
    "config": {
        "on": true,
        "reachable": true
    },
```

Reply:
Ha yes, right, for light it's state/reachable but config/reachable  for sensor.
I think it s from the HUE API, light never have "config" part

Reply:
Thanks for confirmation.